config/settings.yml

```diff
@@ -0,0 +1,7 @@
+oauth2:
```

Comment:
I'm tempted to set these variables in the application via environment variables, because it seems like setting secrets via environment variables makes dockerization and deployment of the application easier. 

Setting the variables via this config file at this point in the application's life is perfectly fine, however.

Any thoughts on this?

Reply:
My goal was to keep config variables secret and to organize them in a clean way.
ENV variables are good, but when you have lots of them, it becomes difficult to manage them.
However, I haven't used the "config" gem with docker, maybe it's not the best match.